test/common.js

```diff
@@ -20,6 +20,11 @@ if (process.env.TEST_THREAD_ID) {
 }
 exports.tmpDir = path.join(exports.testDir, exports.tmpDirName);
 
+// Create tmpDir if it does not exist
+if (! fs.existsSync(exports.tmpDir)) {
+  fs.mkdirSync(exports.tmpDir);
+}
```

Comment:
Well technically `fs.mkdirSync()` could still throw too, for example if permissions are wrong somehow. I doubt it there would be a measurable impact for tests, but even if there was, there's always moving it to a separate function I suppose to limit the scope of the deopt...